tailwind.config.mjs

```diff
@@ -0,0 +1,22 @@
+import tailwindConfig from '@hypothesis/frontend-shared/lib/tailwind.preset.js';
+
+export default {
+  presets: [tailwindConfig],
+  content: [
+    './via/static/scripts/**/*.{js,ts,tsx}',
+    './node_modules/@hypothesis/frontend-shared/lib/**/*.js',
+  ],
+  theme: {
+    extend: {
+      fontFamily: {
```

Comment:
I copied the font family stack here from the LMS app, as I didn't see a default in `@hypothesis/frontend-shared/lib/tailwind.preset.js`. I'm not sure that we want this here though. @lyzadanger can you advise when you're back from 🇧🇪 ?